workflow-ui/backstack-android/src/main/java/com/squareup/workflow1/ui/backstack/ViewStateCache.kt

```diff
@@ -30,6 +32,25 @@ internal constructor(
 ) : Parcelable {
   public constructor() : this(mutableMapOf())
 
+  /**
+   * Cache of the last [Owner] created by [update], so we can restore it when we are restored.
```

Comment:
nit: [Owner] -> [owner][KeyedStateRegistryOwner]?

Reply:
This should have got renamed in the refactor, oops.